code/src/index.css

```diff
@@ -5,9 +5,230 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(24, 24, 20);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  color: #fff;
+  font-size: 22px;
+  margin-bottom: 10%;
+}
+
+h2 {
+  font-family: "Helvetica";
+  font-size: 18px;
+  color: #ffffff;
+  margin-bottom: 0;
+  text-decoration: none;
+}
+
+h3,
+a {
+  font-family: "Helvetica";
+  font-size: 18px;
+  color: #a0a0a0;
```

Comment:
Could these colors be replaced by variables? And other recurring elements such as the Helvetica font-family, too :)

Reply:
Thank you for the suggestion! We will take this ideá with us for the next projects :)